packages/happo-target-firefox/src/waitForImagesToRender.js

```diff
@@ -11,11 +11,24 @@ export function waitForImageToLoad(url) {
 
 export default function waitForImagesToRender() {
   return new Promise((resolve, reject) => {
-    const promises = Array.prototype.slice.call(document.querySelectorAll('img'))
+    const images = Array.prototype.slice.call(document.querySelectorAll('img'));
+    const promises = images
       .map(img => img.src)
       .filter(Boolean)
       .map(waitForImageToLoad);
 
+    images.forEach((img) => {
+      const srcset = img.getAttribute('srcset');
+      if (!srcset) {
+        return;
+      }
+
+      srcset.split(/,\s*/).forEach((entry) => {
```

Comment:
I don't think this will work for urls that contain commas

Reply:
Good catch. Fixed in dc21c33.